lib/moonbeam.js

```diff
@@ -120,6 +120,7 @@ class Moonbeam {
     app.post('/sm-tos', cmw, privRoutes.onPrivateTosMainSetHttpRequest.bind(privRoutes))
     app.post('/gm-tos', cmw, privRoutes.onPrivateTosMainGetHttpRequest.bind(privRoutes))
     app.post('/login', cmw, privRoutes.onPrivateLoginHttpRequest.bind(privRoutes))
+    app.post('/push-tx', cmw, privRoutes.onPrivatePushTxHttpRequest.bind(privRoutes))
```

Comment:
Maybe we should add also integration test case for it?

Reply:
agree here. However, this particular endpoint and the service in general will be reworked massively. Let's get this draft endpoint implementation merged first and implement the tests later on when we have at least some bits of logic inserted.